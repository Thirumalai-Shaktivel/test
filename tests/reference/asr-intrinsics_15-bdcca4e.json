{
    "basename": "asr-intrinsics_15-bdcca4e",
    "cmd": "lfortran --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/../integration_tests/intrinsics_15.f90",
    "infile_hash": "20930d3ef81fa6bac4dbd05caeb3361d84015920f0e02b82bfee86d1",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "asr-intrinsics_15-bdcca4e.stdout",
    "stdout_hash": "db5210cfb750e21cfb64e9a61e26500dc7b59cbf897921d9614110c7",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}