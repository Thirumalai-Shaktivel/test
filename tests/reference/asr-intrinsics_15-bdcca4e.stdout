(TranslationUnit (SymbolTable 1 {intrinsics_15: (Program (SymbolTable 2 {int: (ExternalSymbol 2 int 4 int lfortran_intrinsic_array int Private), x: (Variable 2 x Local (ConstantReal 3.300000 (Real 4 [])) (ConstantReal 3.300000 (Real 4 [])) Parameter (Real 4 []) Source Public Required)}) intrinsics_15 [] [(Print () [(FunctionCall 2 int () [(Var 2 x)] [] (Integer 4 []) (ConstantInteger 3 (Real 4 [])) ())])]), lfortran_intrinsic_array: (Module (SymbolTable 4 {abs: (Function (SymbolTable 5 {abs: (Variable 5 abs ReturnVar () () Default (Integer 4 []) Source Public Required), x: (Variable 5 x In () () Default (Integer 4 [(() ())]) Source Public Required)}) abs [(Var 5 x)] [] (Var 5 abs) Intrinsic Public Interface), allocated: (Function (SymbolTable 6 {allocated: (Variable 6 allocated ReturnVar () () Default (Logical 4 []) Source Public Required), x: (Variable 6 x In () () Default (Integer 4 [(() ())]) Source Public Required)}) allocated [(Var 6 x)] [] (Var 6 allocated) Intrinsic Public Interface), int: (Function (SymbolTable 7 {int: (Variable 7 int ReturnVar () () Default (Integer 4 []) Source Public Required), kind: (Variable 7 kind In () () Default (Integer 4 []) Source Public Required), x: (Variable 7 x In () () Default (Real 4 [(() ())]) Source Public Required)}) int [(Var 7 x) (Var 7 kind)] [] (Var 7 int) Intrinsic Public Interface), lbound: (Function (SymbolTable 8 {dim: (Variable 8 dim In () () Default (Integer 4 []) Source Public Required), lbound: (Variable 8 lbound ReturnVar () () Default (Integer 4 []) Source Public Required), x: (Variable 8 x In () () Default (Integer 4 [(() ())]) Source Public Required)}) lbound [(Var 8 x) (Var 8 dim)] [] (Var 8 lbound) Intrinsic Public Interface), max: (Function (SymbolTable 9 {a: (Variable 9 a In () () Default (Integer 4 []) Source Public Required), b: (Variable 9 b In () () Default (Integer 4 []) Source Public Required), max: (Variable 9 max ReturnVar () () Default (Integer 4 []) Source Public Required)}) max [(Var 9 a) (Var 9 b)] [] (Var 9 max) Intrinsic Public Interface), maxval: (Function (SymbolTable 10 {maxval: (Variable 10 maxval ReturnVar () () Default (Integer 4 []) Source Public Required), x: (Variable 10 x In () () Default (Integer 4 [(() ())]) Source Public Required)}) maxval [(Var 10 x)] [] (Var 10 maxval) Intrinsic Public Interface), min: (Function (SymbolTable 11 {a: (Variable 11 a In () () Default (Integer 4 []) Source Public Required), b: (Variable 11 b In () () Default (Integer 4 []) Source Public Required), min: (Variable 11 min ReturnVar () () Default (Integer 4 []) Source Public Required)}) min [(Var 11 a) (Var 11 b)] [] (Var 11 min) Intrinsic Public Interface), minval: (Function (SymbolTable 12 {minval: (Variable 12 minval ReturnVar () () Default (Integer 4 []) Source Public Required), x: (Variable 12 x In () () Default (Integer 4 [(() ())]) Source Public Required)}) minval [(Var 12 x)] [] (Var 12 minval) Intrinsic Public Interface), real: (Function (SymbolTable 13 {kind: (Variable 13 kind In () () Default (Integer 4 []) Source Public Required), real: (Variable 13 real ReturnVar () () Default (Real 4 []) Source Public Required), x: (Variable 13 x In () () Default (Integer 4 [(() ())]) Source Public Required)}) real [(Var 13 x) (Var 13 kind)] [] (Var 13 real) Intrinsic Public Interface), size: (Function (SymbolTable 14 {size: (Variable 14 size ReturnVar () () Default (Integer 4 []) Source Public Required), x: (Variable 14 x In () () Default (Integer 4 [(() ())]) Source Public Required)}) size [(Var 14 x)] [] (Var 14 size) Intrinsic Public Interface), sum: (Function (SymbolTable 15 {sum: (Variable 15 sum ReturnVar () () Default (Integer 4 []) Source Public Required), x: (Variable 15 x In () () Default (Integer 4 [(() ())]) Source Public Required)}) sum [(Var 15 x)] [] (Var 15 sum) Intrinsic Public Interface), tiny: (Function (SymbolTable 16 {tiny: (Variable 16 tiny ReturnVar () () Default (Real 4 []) Source Public Required), x: (Variable 16 x In () () Default (Integer 4 [(() ())]) Source Public Required)}) tiny [(Var 16 x)] [] (Var 16 tiny) Intrinsic Public Interface), ubound: (Function (SymbolTable 17 {dim: (Variable 17 dim In () () Default (Integer 4 []) Source Public Required), ubound: (Variable 17 ubound ReturnVar () () Default (Integer 4 []) Source Public Required), x: (Variable 17 x In () () Default (Integer 4 [(() ())]) Source Public Required)}) ubound [(Var 17 x) (Var 17 dim)] [] (Var 17 ubound) Intrinsic Public Interface)}) lfortran_intrinsic_array [] .true.)}) [])
